packages/landing/components/sections/HeroBanner/HeroBanner.tsx

```diff
@@ -18,6 +18,35 @@ import { STACKLY_APP_URL } from "@/constants";
 export const HeroBanner = () => {
   return (
     <section className="px-6 pt-16 border-b border-gray-100 md:pt-20">
+      <div className="flex items-center px-3 py-2 mx-auto mb-3 space-x-2 border border-surface-75 rounded-3xl w-fit hover:border-primary-300">
+        <p className="text-xs text-em-med">Live on:</p>
+        <div className="flex items-center space-x-1">
+          <Image
+            src="/assets/images/ethereum-avatar.svg"
+            alt="ethereum logo"
+            width={20}
+            height={20}
+            title="live on ethereum mainnet network"
+            className="hover:scale-125"
+          />
+          <Image
+            src="/assets/images/gnosis-avatar.svg"
+            alt="gnosis logo"
+            width={20}
+            height={20}
+            title="live on gnosis network"
+            className="hover:scale-125"
+          />
+          <Image
+            src="/assets/images/arbitrum-avatar.svg"
+            alt="arbitrum logo"
+            width={20}
+            height={20}
+            title="live on arbitrum one network"
+            className="hover:scale-125"
+          />
```

Comment:
Maybe we could replace this block of code with an array map based on this array (that we can define outside the Component const):
```ts
const liveOnNetworks = [
  { name: "ethereum", title: "ethereum mainnet" },
  { name: "gnosis", title: "gnosis" },
  { name: "arbitrum", title: "arbitrum one" },
];
```

```suggestion
           {liveOnNetworks.map((network) => (
            <Image
              key={network.name}
              src={`/assets/images/${network.name}-avatar.svg`}
              alt={`${network.name} logo`}
              width={20}
              height={20}
              title={`live on ${network.title} network`}
              className="hover:scale-125"
            />
          ))}
```

Reply:
we could but it's okay like this